package.json

```diff
@@ -769,6 +800,14 @@
         {
           "when": "false",
           "command": "SAS.downloadTable"
+        },
+        {
+          "when": "false",
+          "command": "SAS.downloadResource"
+        },
+        {
+          "when": "false",
+          "command": "SAS.uploadResource"
         }
```

Comment:
do we need an entry for saveHTML here?

Reply:
Done in https://github.com/sassoftware/vscode-sas-extension/pull/550/commits/242da852b3b7fa434704cf23f8c561752ab37048